File: FrondEnd/src/Models/Class files/Tutor.js
import User from './User'

export default class Tutor extends User{
    constructor(username, password, studentId, name, email, phone, marks) {
      super(username, password, 'tutor');
      this.studentId = studentId;
      this.name = name;
      this.email = email;
      this.phone = phone;
      this.marks = marks;
    }
}
  
  

File: Backend/src/main/java/com/example/demo/Collection/Tutor.java
package com.example.demo.Collection;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document

public class Tutor extends User {
    private String name;
    @Id
    private String studentID;
    private  String phone;


}
